astropy/modeling/core.py

```diff
@@ -4165,58 +4151,6 @@ def generic_call(self, *inputs, **kwargs):
     return outputs
 
 
-def prepare_bounding_box_inputs(self, input_shape, inputs, bbox):
```

Comment:
The functions removed here are not technically "hidden" but I also don't see them listed explicitly in API doc. Still, might be good to say that they have been removed in the change log.

Reply:
This method is functionally equivalent to this [bound method](https://github.com/WilliamJamieson/astropy/blob/3daa3cdeac6d8a5d197a01844318a893bce9d519/astropy/modeling/bounding_box.py#L439-L471). I.e you can perform the same task by calling `bbox.prepare_inputs(input_shape, inputs)`.

If you would like I can leave the old method in place and just modify it to use the new code?

Reply:
> leave the old method in place and just modify it to use the new code?

Not sure; probably overkill. Might be enough to just mention that they are removed in the change log but @perrygreenfield or @nden should make the final decision.